packages/core-data/src/entities.js

```diff
@@ -213,7 +213,7 @@ async function loadPostTypeEntities() {
 	const postTypes = await apiFetch( {
 		path: '/wp/v2/types?context=view',
 	} );
-	return map( postTypes, ( postType, name ) => {
+	return Object.entries( postTypes ?? {} ).map( ( [ name, postType ] ) => {
```

Comment:
`postTypes` could be `null`, so adding nullish coalescing:

https://github.com/WordPress/gutenberg/blob/f5582817aa2c45ddc7d52dac50025fcfc2a438e3/packages/core-data/src/selectors.ts#L519